samples: bluetooth: rpc_host: update readme (#7055)
nrfconnect/sdk-nrf: samples/bluetooth/rpc_host/README.rst

********

.. note::

>    Currently, the serialization of all Bluetooth APIs is not supported.

Reviewer: This sentence makes me pause when I read it. I suggest softening "all" to "some" for easier reading

```suggestion
   Currently, the serialization of some Bluetooth APIs is not supported.
```

..or using a cliche phrasing, like this:

```suggestion
   Only a subset of Zephyr's Bluetooth APIs are available trough Bluetooth nRF RPC for now.
```

Reviewer 2: Or maybe even something like this:
Currently, only a subset of Zephyr's Bluetooth APIs **is** available over Bluetooth nRF RPC.

Author: I rephrased it as you suggested. Please rereview

>    See the :ref:`ble_rpc_api` of the Bluetooth nRF RPC library to learn about limitations.

Reviewer 2: ```suggestion
   For more details about the limitations, see the :ref:`ble_rpc_api` of the Bluetooth nRF RPC library.
```
